Vagrantfile

```diff
@@ -79,6 +79,24 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     end
   end
 
+  ## Provider-specific settings, libvirt
+  config.vm.provider :libvirt do |v,override|
+    override.vm.box = "fedora21"
+    override.vm.box_url = "http://citozin.com/fedora21.box"
+    override.hostmanager.ignore_private_ip = false
+
+    v.memory = ram_megabytes.to_s
+    v.nested = true
+    v.volume_cache = 'none'
+
+    if Vagrant.has_plugin?("vagrant-cachier")
+      config.cache.scope = :box
+      config.cache.enable :generic, {
```

Comment:
It looks like you just went with the built-in openjdk8. vagrant-cachier is still useful for caching that download, but is there any reason to include this oracle-jdk7 cache directory anymore?